ios/xcode/FluentUITests.xcconfig

```diff
@@ -10,5 +10,7 @@ INFOPLIST_FILE = FluentUI.Tests/Info.plist
 LD_RUNPATH_SEARCH_PATHS = $(inherited) @executable_path/Frameworks @loader_path/Frameworks
 PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.FluentUITests
 PRODUCT_NAME = $(TARGET_NAME)
+SUPPORTED_PLATFORMS = iphoneos iphonesimulator xros xrsimulator
+SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = NO
```

Comment:
Curious - is this needed? What's the default value?

Reply:
Xcode adds this based on what you select in the GUI if you're creating a new scheme. Not sure if they're needed.

Reply:
If you don't include this, the Run Destinations dropdown includes an extra "Apple Vision Pro (Designed for iPad)" entry that we don't want:

<img width="401" alt="VisionProiPad" src="https://github.com/microsoft/fluentui-apple/assets/4934719/ff9d6a38-785e-4ff9-83da-9778b9c94bed">